syft/tensor.py

```diff
@@ -3331,6 +3331,31 @@ def renorm_(self, p, dim, maxnorm):
             self.data = syft.math.renorm(self, p, dim, maxnorm).data
             return self
 
+    def stride(self, dim=None):
+        """
+        Returns the jump necessary to go from one element to the next one in the specified dimension dim.
+
+        Parameters
+        ----------
+        dim : dimension
+            The first operand in the stride operation
+
+        Returns
+        -------
+        Tuple
```

Comment:
A tuple is returned when there's no Argument, If not an int is returned.
so maybe you can change the doc to reflect that?